packages/flutter/test/material/input_chip_test.dart

```diff
@@ -296,29 +294,50 @@ void main() {
     focusNode2.dispose();
   });
 
-  testWidgets('Input chip check mark color is determined by platform brightness when light', (WidgetTester tester) async {
+  testWidgets('Material2 - Input chip check mark color is determined by platform brightness when light', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData(useMaterial3: false);
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(),
+      useMaterial3: theme.useMaterial3,
     );
 
-    expectCheckmarkColor(
-      find.byType(InputChip),
-      Colors.black.withAlpha(0xde),
+    expectCheckmarkColor(find.byType(InputChip), Colors.black.withAlpha(0xde));
+  });
+
+  testWidgets('Material3 - Input chip check mark color is determined by platform brightness when light', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData();
+    await pumpCheckmarkChip(
+      tester,
+      chip: selectedInputChip(),
+      useMaterial3: theme.useMaterial3,
     );
+
+    expectCheckmarkColor(find.byType(InputChip), Colors.black.withAlpha(0xde));
   });
 
-  testWidgets('Input chip check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
+  testWidgets('Material2 - Input chip check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData(brightness: Brightness.dark, useMaterial3: false);
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(),
-      brightness: Brightness.dark,
+      brightness: theme.brightness,
```

Comment:
nit: I think it's pretty straightforward to just set `brightness: Brightness.dark, useMaterial3: false` because we don't need to use the `theme` object in expect().

Reply:
Makes sense!